docs/content/develop/templates.md

```diff
@@ -0,0 +1,280 @@
++++
+title = "Templates & slots"
+description = ""
+weight = 3
+draft = false
+bref = ""
+toc = true
+menu = "start"
+tags = [ "start" ]
++++
+
+
+[TOC]
+
+# Light DOM vs. Shadow DOM
+
+The concept of light DOM is already familiar to any web developer. Basic HTML elements that you place into a `.html` file are light DOM. Tags like `<h1>`, `<div>` and `<aside>` are all simple light DOM. 
+
+However, when you add a tag to the light DOM such as `select`, the browser has additional styles that are invoked from within the Shadow DOM. 
+
+When building web components, you may choose whether to keep content in the light DOM (i.e. pfe-tab), or to copy it from the light DOM into the shadow DOM (i.e. pfe-accordion-header), or to place it in the shadow DOM only (i.e. close button on pfe-modal).
+
+Consider the pros and cons for each approach, and remember that web components should still contain semantic markup and fail gracefully, so that users or search crawlers can still understand the content on the page.
+
+
+## Light DOM
+
+
+```
+  <select>
+    <option value="books">Books</option>
+    <option value="html">HTML</option>
+    <option value="css">CSS</option>
+    <option value="php">PHP</option>
+    <option value="js">JavaScript</option>
+  </select>
+```   
+
+ <select>
```

Comment:
Should this be duplicative of the above?

Reply:
yep! one is a code block, one is a rendered HTML element